indy-utils/src/main/kotlin/com/luxoft/blockchainlab/hyperledger/indy/Models.kt

```diff
@@ -103,15 +106,15 @@ data class KeyCorrectnessProof(val c: String, val xzCap: String, val xrCap: List
 data class Claim(
         val schemaId: String,
         val credDefId: String,
-        val revReg: String?,
+        val revReg: RawJsonMap?,
```

Comment:
Why revReg is nullable? The type should be more specific  than RawJsonMap

Reply:
revReg is nullable because it actually can contain null as a value when revocation is disabled. RawJsonMap is used here because revReg contains revocation registry state at the moment a claim was created, which we don't need to bother until we use ledger as a storage.